modules/cnn_3dobj/test/test_cnn_3dobj_feature_extract.cpp

```diff
@@ -30,7 +30,7 @@ void CV_CNN_Feature_Test::run(int)
     String network_forIMG = cvtest::TS::ptr()->get_data_path() + "3d_triplet_testIMG.prototxt";
     String mean_file = "no";
     std::vector<String> ref_img;
-    String target_img = String(ts->get_data_path()) + "1_8.png";
+    String target_img = String(ts->get_data_path()) + "4_78.png";
```

Comment:
Ok, lets wait for https://github.com/opencv/opencv/pull/7750. It allows to add different testdata sources using `ocv_add_testdata` macro in CMake.